src/controllers/expenses.contr.js

```diff
@@ -0,0 +1,68 @@
+const expensesServ = require('../services/expenses.service');
+const usersServ = require('../services/users.service');
+const statusCode = require('../utils/statusCode');
+
+const getAllExpenses = (req, res) => {
```

Comment:
You use like `expensesContr.[your method]` so we already know that it related to expenses
```suggestion
const getAll = (req, res) => {
```